app/scripts/controllers/permissions/index.js

```diff
@@ -23,16 +24,24 @@ import {
   CAVEAT_TYPES,
 } from './enums'
 
+// instanbul ignore next
+const noop = () => undefined
+
+/**
+ * @class
+ * @constructor
+ * @public
+ */
```

Comment:
What is this for? Isn't it implied by the `export` and the `class` keyword that this is a public class? And it's not even a `constructor` really.

Reply:
I'd love to tell you what's going on here, but I don't know. It's removed now, lol.